### PR TITLE
feat: add dat-gui based CSS debug menu

### PR DESCRIPTION
**Pull request by thorseraq**
https://user-images.githubusercontent.com/20554850/215510465-af4b2116-921e-4d10-93ec-a3b924ac36e4.mov

### CODE REVIEW

File: packages/blocks/src/page-block/default/default-page-block.ts [hunk 1]
```diff
@@ -79,6 +79,10 @@ export class DefaultPageBlockComponent
       overflow-y: auto;
       height: 100%;
     }
+
+    .dg.ac {
```
Comment (doodlewind) [inline packages/blocks/src/page-block/default/default-page-block.ts:83]: Can we move this style into `DebugMenu`?

Comment (thorseraq) [reply packages/blocks/src/page-block/default/default-page-block.ts:83]: The css scope in DebugMenu can't affect on dat-gui element🫠

Comment (thorseraq) [reply packages/blocks/src/page-block/default/default-page-block.ts:83]: There seems can be a way to work

Comment (doodlewind) [reply packages/blocks/src/page-block/default/default-page-block.ts:83]: You can inject the style globally from JavaScript side in `DebugMenu`, which can be also better than hard-coded here.

**Review by doodlewind — APPROVED** [review]
LGTM, thanks for bringing better debug DX!

**Review by Himself65 — APPROVED** [review]

**Pull request merged**